test/integration/wdio.conf.js

```diff
@@ -196,21 +193,19 @@ function makeCapabilities(config) {
         case 'chromedriver':
             return (() => {
                 const args = [
-                    '--disable-gpu',
-                    '--no-sandbox',
+                    'headless',
+                    'disable-gpu',
                     `window-size=${config.WIDTH},${config.HEIGHT}`,
                 ];
-                if (config.HEADLESS === 't') {
-                    args.push('--headless');
-                }
                 return {
                     browserName: 'chrome',
+                    browserVersion: '128',
                     acceptInsecureCerts: true,
                     maxInstances: 1,
                     'goog:chromeOptions': {
                         args,
-                        binary: CHROME_BINARY,
                     },
+                    'wdio:enforceWebDriverClassic': true
```

Comment:
The latest wdio uses the w3-specced webdriver BiDi protocol. There's something strange with either our configuration, or the mash of versions of things, that prevents that protocol from loading pages from our app. So forcing it back to webDriverClassic fixes that (at least, until they ditch that all together).